hudi-common/src/main/java/org/apache/hudi/common/fs/FSUtils.java

```diff
@@ -276,6 +276,13 @@ public static void processFiles(FileSystem fs, String basePathStr, Function<File
     }
   }
 
+  public static FileStatus[] getFilesInPartition(HoodieEngineContext engineContext, HoodieMetadataConfig metadataConfig,
+                                                 String basePathStr, Path partitionPath) throws IOException {
+    HoodieTableMetadata tableMetadata = HoodieTableMetadata.create(engineContext, metadataConfig, basePathStr,
```

Comment:
needs to be closed. otherwise can leak connections.

Reply:
Thanks for remind me this!